patches/server/0843-Add-Alternate-Current-redstone-implementation.patch

```diff
@@ -0,0 +1,2688 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Simon Gardling <titaniumtown@gmail.com>
```

Comment:
IMO SpaceWalkerRS should be credited as the patch author, with you as the co-author, similar to how egg did it [when porting eigencraft](https://github.com/PaperMC/Paper/pull/1494/files#diff-67d9f633149ac5490b2de8cd95b713ef29bd2749a41e022a40a23babcefe13a1R2-R7) (further discussion [here](https://github.com/PaperMC/Paper/pull/1494#discussion_r221082930))

Reply:
Good point! I'll do that after I finished addressing some concerns that SpaceWalker brought up. Thanks!

Reply:
Fixed!